keyboards/dztech/dz60rgb/keymaps/xunz/rules.mk

```
@@ -0,0 +1,6 @@
+NKRO_ENABLE = yes                 # USB Nkey Rollover
+# VELOCIKEY_ENABLE = yes
+
+EXTRAFLAGS += -flto
```

You should use: 

```suggestion
LINK_TIME_OPTIMIZATION_ENABLE = yes
```

This will actually automatically add the "no action macro" and "no action function"  defines, as well.